apps/music/style/music.css

```diff
@@ -266,12 +290,15 @@ body.picker-mode #title-player {
   overflow-y: auto;
   background: -moz-linear-gradient(center top, #303030 0%, #000000 100%) repeat scroll 0 0 transparent;
   z-index: 10;
-  transition: transform 300ms ease;
+  transition: transform 300ms ease, visibility 300ms;
+  visibility: hidden;
 }
 
 .view {
   width: 100%;
   height: auto;
+  visibility: hidden;
+  transition: visibility 300ms;
```

Comment:
Not sure what you mean. You want to get rid of the animations? That would require some UX changes, no?